src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyPageData.java

```diff
@@ -7,6 +7,8 @@
 
 public class InstructorFeedbackEditCopyPageData extends PageData {
     public List<CourseAttributes> courses;
+    public String courseId;
+    public String fsName;
```

Comment:
Same as my previous comment. I prefer private too but for now I'm following the existing pattern.